cellprofiler/modules/identifyprimaryobjects.py

```diff
@@ -79,22 +79,25 @@
    grayscale using the **ColorToGray** module.
 -  If your images are brightfield/phase/DIC, they may be processed with the
    **EnhanceOrSuppressFeatures** module with its "*Texture*" or "*DIC*" settings.
-   You may also want to check our `tutorial`_ on preprocessing these images with
+-  If you struggle to find effective settings for this module, you may
+   want to check our `tutorial`_ on preprocessing these images with
    ilastik prior to using them in CellProfiler.
 
 What are the advanced settings?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 **IdentifyPrimaryObjects** allows you to tweak your settings in many ways;
-so many so that it can often become confusing where you should start.  
+so many that it can often become confusing where you should start. This is
+typically the most important but complex step in creating a good pipeline,
+so do not be discouraged: other modules are easier to configure!
 Using **IdentifyPrimaryObjects** with *'Use advanced settings?'* set to *'No'* 
-allows you to quickly try to segment your objects based only the size and input
-image; CellProfiler will then use its built-in defaults to decide how to set the 
-threshold and how to break clumped objects apart.  If you are happy with the 
-segmentation produced by the default settings, you can then move on to 
+allows you to quickly try to identify your objects based only their typical size;
+CellProfiler will then use its built-in defaults to decide how to set the 
+threshold and how to break clumped objects apart. If you are happy with the 
```

Comment:
@AnneCarpenter "If you are unhappy"?

Reply:
Nope, "happy". I'm always trying to make ppl happy but in this case it's justified as is :)

Reply:
Cool! I wanted to double check. :)